lib/matplotlib/tests/test_axes.py

```diff
@@ -2815,6 +2815,16 @@ def test_errobar_nonefmt():
         assert np.all(errbar.get_color() == mcolors.to_rgba('C0'))
 
 
+@image_comparison(baseline_images=['errorbar_with_prop_cycle'],
+                  extensions=['png'])
```

Comment:
Can this get a `style=mpl20` argument so it uses the new style?

Reply:
yes. thanks for the nudge. i can't the docs to build properly on my machine at home so this has stalled out a bit.